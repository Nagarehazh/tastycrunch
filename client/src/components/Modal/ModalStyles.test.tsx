import renderer from 'react-test-renderer'
import 'jest-styled-components'
import {
    Overlay,
    ModalContainer,
    EncabezadoModal,
    ButtonClose,
} from './ModalStyles'


describe('Modal styles render correctly', () => {
    it('Overlay renders correctly', () => {
        const tree = renderer.create(<Overlay />).toJSON()
        expect(tree).toMatchSnapshot()
    })
    it('ModalContainer renders correctly', () => {
        const tree = renderer.create(<ModalContainer />).toJSON()
        expect(tree).toMatchSnapshot()
    })
    it('EncabezadoModal renders correctly', () => {
        const tree = renderer.create(<EncabezadoModal />).toJSON()
        expect(tree).toMatchSnapshot()
    })
    it('ButtonClose renders correctly', () => {
        const tree = renderer.create(<ButtonClose />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})

