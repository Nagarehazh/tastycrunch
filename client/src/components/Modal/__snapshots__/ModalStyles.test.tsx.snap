// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal styles render correctly ButtonClose renders correctly 1`] = `
.c0 {
  background: transparent;
  border: none;
  font-size: 1.6rem;
  font-weight: 500;
  color: white;
  cursor: pointer;
  position: absolute;
  top: 20px;
  right: 20px;
  width: 30px;
  height: 30px;
  border: none;
  border-radius: 5px;
  -webkit-transition: all 0.3s ease;
  transition: all 0.3s ease;
}

.c0:hover {
  background: #F2F2F2;
  color: #000;
}

<button
  className="c0"
/>
`;

exports[`Modal styles render correctly EncabezadoModal renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 20px;
  padding-bottom: 20px;
  border-bottom: 1px solid #E8E8E8;
}

.c0 h3 {
  font-weight: 500;
  font-size: 1.6rem;
  color: white;
}

<div
  className="c0"
/>
`;

exports[`Modal styles render correctly ModalContainer renders correctly 1`] = `
.c0 {
  width: 500px;
  min-height: 100px;
  position: relative;
  border-radius: 5px;
  box-shadow: rgba(100,100,111,0.2) 0px 7px 29px 0px;
  padding: 20px;
  background: linear-gradient(-45deg,#ee7752,#e73c7e,#23a6d5,#23d5ab);
  background-size: 400% 400%;
  -webkit-animation: gradient 15s ease infinite;
  animation: gradient 15s ease infinite;
}

<div
  className="c0"
/>
`;

exports[`Modal styles render correctly Overlay renders correctly 1`] = `
.c0 {
  width: 100vw;
  height: 100vh;
  position: fixed;
  top: 0;
  left: 0;
  background-color: rgba(0,0,0,0.5);
  padding: 40px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  z-index: 5000;
}

<div
  className="c0"
/>
`;
